packages/mongo/mongo.d.ts

```diff
@@ -60,7 +60,7 @@ export namespace Mongo {
          * The server connection that will manage this collection. Uses the default connection if not specified. Pass the return value of calling `DDP.connect` to specify a different
          * server. Pass `null` to specify no connection. Unmanaged (`name` is null) collections cannot specify a connection.
          */
-        connection?: Object | null | undefined;
+        connection?: DDP.DDPStatic | null | undefined;
```

Comment:
Here was missing a `import { DDP } from 'meteor/ddp';` will be added before the next rc